README.md.tpl

```diff
@@ -229,3 +229,90 @@ This package is well tested and used in many projects. The API should be conside
 
 Feel free to create an [issue](https://github.com/dave/dst/issues) or chat in the 
 [#dst](https://gophers.slack.com/messages/CCVL24MTQ) Gophers Slack channel.
+
+
+## Building/Developing/Contributing
+
+The `dst` package relies heavily on code generation. So heavily, in fact, that *this* `README.md` is
+itself generated by a preprocessor! Here are a few tips/tricks to get started developing `dst`.
+
+### Adding Features, Fixing Bugs
+
+Generally speaking, code handling node annotation is generated from the files in the `gendst` directory.
+In order to add/change decorations, edit the `gendst/data.go` file. Once you have made your edits, you can
+regenerate the library's source code based on those changes by
+
+1. `go build gendst/*.go`: This command will generate a `clone` binary in the root directory of the folder
```

Comment:
Probably better to do: `go run github.com/dave/dst/gendst` which compile and run the `gendst` command... (The tool shouldn't be called `clone`... I guess it's because the first file in the `*.go` wildcard results is `clone.go`)

Reply:
Absolutely! I honestly had no idea how to get it to work when I first started development, so this was the best I could come up with! I hope that this was helpful!